### DISCUSSION TITLE
Modifying the given columns and tables

### DISCUSSION THREAD

**Discussion by meatandfat** [vendurehq/vendure#2966] [Q&A] [answered]
Hi. Is there a recommended way to modify the built-in columns/tables? For example, <s>postgres recommends using "money" for currency columns. And</s> I'd like to change whether or not nulls are allowed and add indexes and unique constraints

(Sorry, if this is mentioned somewhere. It didn't come up in any of my searches)

**Comment by michaelbromley** [comment, marked as answer]
Hi, this is possible via this API: https://docs.vendure.io/reference/typescript-api/configuration/entity-options/#entitymetadatamodifier

**Reply by meatandfat** [reply]
Thank you
